Makefile

```diff
@@ -346,7 +346,7 @@ kplutus-pyk:
 # Installing
 # ----------
 
-install_bins := kplc kplutus-covr
+install_bins := kplc kplutus-covr contract-compile contract-prove
```

Comment:
Could we make these into kplc commands instead? ie. `kplc contract-compile` and `kplc contract-prove`, so as not to put executables on the user's PATH that look irrelevant.

Reply:
Makes sense, even though I think the commands are relevant.

Reply:
The names of the executables don't reference KPlutus in any way, that was my concern. In either case, I think adding CLI functionality means adding functionality to `kplc`.